SharePointFramework/PortfolioWebParts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pp365-portfoliowebparts",
-  "version": "1.5.4",
+  "version": "1.5.4-749.2",
```

Comment:
@Remi749 What is 749? Issue key? PR key?

Reply:
PR key preferably @Remi749

Reply:
But it's not super important. It just gives some context.

Reply:
Ait, I'll keep that in mind 👍